src/sx-design/src/__tests__/Heading.test.js

```diff
@@ -0,0 +1,79 @@
+// @flow
+
+/* global document */
+
+import * as React from 'react';
+import { render, unmountComponentAtNode } from 'react-dom';
+import prettier from 'prettier';
+
+import Heading from '../Heading';
+import Section from '../Section';
+
+// https://reactjs.org/docs/testing-recipes.html
+
+let container: HTMLDivElement | null = null;
+beforeEach(() => {
+  // setup a DOM element as a render target
+  container = document.createElement('div');
+  document.body?.appendChild(container);
```

Comment:
why not use testing-library/react instead, that does all this setup and teardown for you?